src/backend/app/projects/project_crud.py

```diff
@@ -58,96 +52,62 @@ async def create_project_with_project_info(
         },
     )
 
-    if not new_project_id:
+    if not project_id:
         raise HTTPException(status_code=500, detail="Project could not be created")
     # Fetch the newly created project using the returned ID
     select_query = f"""
         SELECT id, name, short_description, description, per_task_instructions, outline
         FROM projects
-        WHERE id = '{new_project_id}'
+        WHERE id = '{project_id}'
     """
     new_project = await db.fetch_one(query=select_query)
     return new_project
 
 
 async def get_project_by_id(
-    db: Session = Depends(database.get_db), project_id: Optional[int] = None
-) -> db_models.DbProject:
-    """Get a single project by id."""
-    db_project = (
-        db.query(db_models.DbProject)
-        .filter(db_models.DbProject.id == project_id)
-        .first()
-    )
-    return await convert_to_app_project(db_project)
+    db: Database, author_id: uuid.UUID, project_id: Optional[int] = None
+):
+    """Get a single project &  all associated tasks by ID."""
+    raw_sql = """
+    SELECT
+        projects.id,
+        projects.name,
+        projects.short_description,
+        projects.description,
+        projects.per_task_instructions,
```

Comment:
More of a db schema related question, but I don't understand this variable name. 

Isn't it just the instructions for the project?
It could probably just be `projects.instructions`?

Reply:
@spwoodcock  I am returning only the necessary fields of the projects model instead of the whole model, and those fields are also validated using Pydantic schemas.

Regarding the variable name, I agree it could probably be just `projects.instructions` instead of `projects.per_task_instructions`.